examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -38,6 +38,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		26D36078B4738B64685A0B6F /* Pods-Runner.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Runner.release.xcconfig"; path = "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"; sourceTree = "<group>"; };
```

Comment:
Let CocoaPods CocoaPod.

Reply:
Interestingly enough I observed cocoapods changing these files when I was experimenting with the xcode build changes and assumed I was doing something wrong. Sounds like that isn't the case?

Reply:
The SRCROOT -> PODS_ROOT change was
https://github.com/CocoaPods/CocoaPods/commit/f8c09f2b0431f5b82768bc5477b06503899fb152 which looks like it was added in CocoaPods 1.6.0beta1.
The shell script build phase comment was added with CocoaPods 1.3.0beta2
https://github.com/CocoaPods/CocoaPods/commit/f8de0f8a72cb9c5f765638666edf3eb7f5e280ef#diff-241ce887591396cd854a10e604ce5ef6R105
Not sure where the xcconfig additions to the Pod group came from, but I assume also some CocoaPods change.

Reply:
These changes aren't really important, but it would be nice to locally run those build tests and not have any working changes.